src/commands/mobile_app/upload.rs

```diff
@@ -143,41 +144,50 @@ pub fn execute(matches: &ArgMatches) -> Result<()> {
             sha.as_deref(),
             build_configuration,
         ) {
-            Ok(_) => {
+            Ok(artifact_id) => {
                 info!("Successfully uploaded file: {}", path.display());
-                uploaded_paths.push(path.to_path_buf());
+                uploaded_paths_and_ids.push((path.to_path_buf(), artifact_id));
             }
             Err(e) => {
                 debug!("Failed to upload file at path {}: {}", path.display(), e);
-                errored_paths.push(path.to_path_buf());
+                errored_paths_and_reasons.push((path.to_path_buf(), e));
             }
         }
     }
 
-    if !errored_paths.is_empty() {
+    if !errored_paths_and_reasons.is_empty() {
         warn!(
             "Failed to upload {} file{}:",
-            errored_paths.len(),
-            if errored_paths.len() == 1 { "" } else { "s" }
+            errored_paths_and_reasons.len(),
+            if errored_paths_and_reasons.len() == 1 {
+                ""
+            } else {
+                "s"
+            }
         );
-        for path in errored_paths {
-            warn!("  - {}", path.display());
+        for (path, reason) in errored_paths_and_reasons {
+            warn!("  - {} ({})", path.display(), reason);
         }
     }
 
-    println!(
-        "Successfully uploaded {} file{} to Sentry",
-        uploaded_paths.len(),
-        if uploaded_paths.len() == 1 { "" } else { "s" }
-    );
-    if uploaded_paths.len() < 3 {
-        for path in &uploaded_paths {
-            println!("  - {}", path.display());
-        }
-    }
-
-    if uploaded_paths.is_empty() {
+    if uploaded_paths_and_ids.is_empty() {
         bail!("Failed to upload any files");
+    } else {
+        println!(
+            "Successfully uploaded {} file{} to Sentry",
+            uploaded_paths_and_ids.len(),
+            if uploaded_paths_and_ids.len() == 1 {
+                ""
+            } else {
+                "s"
+            }
+        );
+        if uploaded_paths_and_ids.len() < 3 {
+            for (path, artifact_id) in &uploaded_paths_and_ids {
+                let url = format!("{base_url}/{org}/preprod/{project}/{artifact_id}");
```

Comment:
I think it might be better if we adjust the endpoint on the server side to return the full analysis URL. That way, this code remains future-compatibile, in case a future version of Sentry changes the URL for analyzing an artifact.

Although, if we are reasonably confident we will not make such a change, and something like this could be mitigated with redirects, I think it is okay to reconstruct the URL in the CLI